pallets/runtime/common/src/fee_details.rs

```diff
@@ -0,0 +1,27 @@
+use codec::{Decode, Encode};
```

Comment:
The filename should be different, It doesn't reflect what it should be as per the file name.